code/modules/hydroponics/biogenerator.dm

```diff
@@ -195,86 +195,91 @@
 	. += emissive_appearance(icon, "[icon_state]_o_screen", src)
 
 /obj/machinery/biogenerator/wrench_act(mob/living/user, obj/item/tool)
-	. = ..()
-	default_unfasten_wrench(user, tool)
-	return ITEM_INTERACT_SUCCESS
+	if(default_unfasten_wrench(user, tool))
+		return ITEM_INTERACT_SUCCESS
+	return ITEM_INTERACT_BLOCKING
```

Comment:
```suggestion
	switch(default_unfasten_wrench(user, tool))
		if(SUCCESSFUL_UNFASTEN)
			return ITEM_INTERACT_SUCCESS
		if(FAILED_UNFASTEN)
			return ITEM_INTERACT_BLOCKING
	return NONE
```
`default_unfasten_wrench` can return `CANT_UNFASTEN` which is expected to not cancel attack chain